src/utils/commands.js

```diff
@@ -15,8 +15,8 @@ export const registerCommandModule = (name, command, handler) => {
 
 export const registerGuildCommands = async () => {
   // TODO: Replace these with ENV variables
-  const clientId = '873205734228234290';
-  const guildId = '864894667774361623';
+  const clientId = process.env.CLIENT_ID;
```

Comment:
This is great, but in the future you should try to keep unrelated changes relegated to separate commits.